react-web/src/pages/subscription/history/SubscriptionHistory.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { useEffect, useState } from "react";
+import { useNavigate } from "react-router-dom";
+import dayjs from "dayjs";
+import utc from "dayjs/plugin/utc";
+import tz from "dayjs/plugin/timezone";
+
+import TableComponent from "components/Table/Table";
+import { fetchData } from "api/api";
+import { Subscription } from "../Subscription.interface";
+import Toast from "components/Toast/Toast";
+
+dayjs.extend(utc)
+dayjs.extend(tz)
+
+const SubscriptionHistory = () => {
+    const navigate = useNavigate();
+    const [data, setData] = useState<Array<Subscription>>([]);
+    const [errorToast, setErrorToast] = useState<{display: boolean; statusText?: string; message?: string;}>({display: false});
+    const timeZone = dayjs.tz.guess()
+
+    // to be invoked only once at first
+    useEffect(() => {
+      fetchTableData();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, []);
+
+    const fetchTableData = async () => {
+      try {
+        const result = await fetchData.get("/profile/current/subscriptions?just-enabled=false")
+        if (result.data.length) {
+          setData(result.data);
+        }
+      } catch (err) {
+        handleError(err)
+      }
+    };
+
+    const handleError = (error: any) => {
+      if (error.response) {
+        setErrorToast({display: true, statusText: error.response.statusText, message: error.response.data || undefined})
+      } else {
+        setErrorToast({display: true})
+      }
+      const timeout = setTimeout(() => { clearTimeout(timeout); setErrorToast({display: false}) }, 3000)
+    }
+
+    const columns = React.useMemo(() => [
+        {
+            Header: "Type of Subscription",
+            accessor: "name",
+        },
+        // {
+        //     Header: "Amount Paid(in ADA)",
+        //     accessor: "price",
+        //     Cell: (props: any) => (
+        //         <span>{(props.row.original.price / 1000000).toFixed(2)}</span>
+        //     )
+        // },
+        {
+            Header: "Amount Paid",
+            accessor: "adaUsdPrice",
+            Cell: (props: any) => (
+                <span>${Math.floor((props.row.original.price/1000000) * props.row.original.adaUsdPrice)}</span>
```

Comment:
Let's properly round this as a fiat.
```suggestion
                <span>${Math.round((props.row.original.price/1000000) * props.row.original.adaUsdPrice * 100) / 100}</span>
```